aws/resource_aws_wafv2_web_acl_test.go

```diff
@@ -1538,6 +1706,166 @@ resource "aws_wafv2_web_acl" "test" {
 `, name, countryCodes)
 }
 
+func testAccAwsWafv2WebACLConfig_CustomRequestHandling_Count(name, firstHeader string, secondHeader string) string {
+	return fmt.Sprintf(`
+resource "aws_wafv2_web_acl" "test" {
+  name        = "%[1]s"
+  description = "%[1]s"
+  scope       = "REGIONAL"
+
+  default_action {
+    allow {}
+  }
+
+  rule {
+    name     = "rule-1"
+    priority = 1
+
+    action {
+      count {
+        custom_request_handling {
+          insert_headers {
+            name  = "%[2]s"
+            value = "test-value-1"
+          }
+
+          insert_headers {
```

Comment:
@anGie44 I had a naming question for you. Should this attribute be `insert_headers` or `insert_header`? I've seen both used in various terraform resources for blocks that can repeat. I would like to make sure I'm choosing the most preferred option going forward.

Reply:
good question! should be singular similar to the `rule` block .. for this repeatable block, a `TypeSet` may be better suited if order is not important or if there's no guarantee on order when returned from the AWS API

Reply:
looks like `response_headers` needs an update as well

Reply:
Sounds good. I just pushed changes that make both `insert_header` and `response_header` named singular and changed them to `TypeSet`.